docs/guides/tilgettingstarted.md

```diff
@@ -0,0 +1,135 @@
+The TriggerMesh Integration Language (TIL) is a configuration language based on the [HCL syntax](hcl-spec) which purpose is to provide a user-friendly interface for describing [TriggerMesh Bridges]().
+
+!!! info "Use TIL with the TriggerMesh Platform"
+    The TriggerMesh Integration Language is easily usable with the `til` CLI which helps you to generate the manifests describing your event-driven application. It is the perfect companion to the TriggerMesh controller. Go to the [complete documentation](../til/Introduction.md) for the detailed specification.
+
+## Installation
+
+Download the CLI from the GitHub [release page](https://github.com/triggermesh/til/releases). For example on Linux x86_64 architecture do:
+
+```console
+wget https://github.com/triggermesh/til/releases/download/v0.1.0/til_v0.1.0_linux_amd64
+```
+
+Put it in your PATH and make the binary executable
+
+```console
+mv til_v0.1.0_linux_amd64 /usr/local/bin/til
+chmod +x /usr/local/bin/til
+```
+
+Check that you can run the command
+
+```console
+$ til -h
+Interpreter for TriggerMesh's Integration Language.
+
+USAGE:
+    ./til <command>
+
+COMMANDS:
+    generate     Generate Kubernetes manifests for deploying a Bridge.
+    validate     Validate a Bridge description.
+    graph        Represent a Bridge as a directed graph in DOT format.
+```
+
+For the complete usage reference go to the [TIL documentation](../til/Introduction.md)
+
+## Example
+
+With the `til` CLI installed you are ready to write your first Bridge.
+
+In this guide we take the simple example of a point to point conection between a GitHub repository and an AWS Lambda function. This assumes:
+
+* We are using a sample repository on Github called `sebgoa/transform`. Pick one that you have access to
+* We are using a GitHub token stored as a Kubernetes secret called `my-github-secret`
+* We are using AWS IAM credentials stored as a Kubernettes secret called `my-aws-access-keys`
+* We are using an AWS Lambda function called `github-processing` as our Target.
+* We want to receive all events related to `issues` in our repository.
+
+Using the TIL syntax you write this point to point connection in a file called `til-demo.tf` as is:
+
+```
+source "github" "git_source" {
+  owner_and_repository = "sebgoa/transform"
+  event_types = ["issues"]
+  tokens = secret_name("my-github-secret")
+
+  to = target.my_lambda
+}
+
+target aws_lambda "my_lambda" {
+  arn = "arn:aws:lambda:us-east-1:123456789:function:github-processing"
+  credentials = secret_name("my-aws-access-keys")
+}
+```
```

Comment:
And before I forget, we should enable syntax highlighting:

```` ```hcl ````

Reply:
We would probably need [this](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#configuration) + some custom CSS.

But my point here was just that we need to annotate the code block with ```` ```hcl ```` instead of just ```` ``` ````.

Reply:
annotating it is easy, but what's the point if there is no lexer for it https://pygments.org/docs/lexers/

Reply:
2 reasons:
- Maybe we'll have one at some point in the future, but by then we'll all have forgotten about this one snippet, whereas adding it now takes no effort.
- The language will (should) be added as a `class` attribute on the generated HTML block, which is good for semantics, accessibility, ..., you name it.